streaming/src/test/java/org/apache/spark/streaming/Java8APISuite.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package test.org.apache.spark.java8.dstream;
+package org.apache.spark.streaming;
```

Comment:
Is it worth it to keep the "test." package there? That avoids having the test use package-private APIs inadvertently (if any of those exit).